implementation/src/main/java/io/smallrye/openapi/runtime/scanner/OpenApiAnnotationScanner.java

```diff
@@ -449,6 +456,31 @@ private void processJaxRsResourceClass(OpenAPIImpl openApi, ClassInfo resourceCl
         }
     }
 
+    /**
+     * Build a map between exception class name and its corresponding @ApiResponse annotation in the jax-rs exception mapper
+     *
+     */
+    private Map<DotName, AnnotationInstance> processExceptionMappers() {
+        Map<DotName, AnnotationInstance> exceptionHandlerMap = new HashMap<>();
+        Collection<ClassInfo> exceptionMappers = this.index
+                .getKnownDirectImplementors(OpenApiConstants.DOTNAME_EXCEPTION_MAPPER);
+
+        for (ClassInfo classInfo : exceptionMappers) {
+            DotName exceptionDotName = classInfo.interfaceTypes().get(0).asParameterizedType().arguments().get(0).name();
+
+            MethodInfo toResponseMethod = classInfo.method("toResponse", Type.create(exceptionDotName, Type.Kind.CLASS));
```

Comment:
This is neat - users can annotate the `toResponse` method with `@ApiResponse`.  I like it. :)